src/your-first-flutter-app-to-appstore/index.md

```diff
@@ -182,37 +177,56 @@ Once your build successfully completes, you will have an **unsigned** artifact t
+* ***Ad-Hoc Provisioning Profiles*** — Used for distributing an app to a small number of users (or testers) directly. In this case, you need to add the device UDIDs manually into the Apple Developer portal.
+* ***App Store Provisioning Profile*** — Used for uploading your app build to App Store Connect for distribution via TestFlight or the public App Store.
 
-Luckily for the purposes of this tutorial, most of the details of dealing with obtaining, managing and using Provisioning profiles can be handled by Codemagic via the App Store Connect API. But in order to allow Codemagic to do so on behalf of your app, you first need to obtain an App Store Connect API **key**. The process to do so is:
+#### **How Codemagic Simplifies Code Signing** ####
 
-1. Log in to App Store Connect and navigate to Users and Access &gt; Keys.
-2. Click on the + sign to generate a new API key.
-3. Enter the name for the key and select an access level. We recommend choosing App Manager, considering that Developer does not have the required permissions to upload to the store. Read more about Apple Developer Program role permissions [here](https://developer.apple.com/help/app-store-connect/reference/role-permissions/).
-4. Click Generate.
-5. As soon as the key is generated, you can see it added to the list of active keys. Click Download API Key to save the private key for later. Note that the key can only be downloaded once.
+Fortunately, for this tutorial, most of the complexities of obtaining, managing, and using provisioning profiles can be handled by Codemagic through the App Store Connect API.
 
-This process is also covered in   [this section of the Codemagic documentation](https://docs.codemagic.io/flutter-code-signing/ios-code-signing/#step-1-creating-an-app-store-api-key-for-codemagic).  Once you have obtained an API key, you will need to enter it, along with the **`Issuer ID`** and **`Key ID`** both of which can be found on the App Store Connect website:
+However, to allow Codemagic to manage this on behalf of your app, you first need to obtain an ***App Store Connect API key***. The process to do so is as follows:
+
+1. Log in to App Store Connect and navigate to **Users and Access &gt; Integrations**.
+2. Click on the "+" sign on the **Team Keys** tab to generate a new API key.
+3. Enter a name for the key and select an access level.
+* We recommend choosing App Manager, as the Developer role does not have the required permissions to upload to the store. Read more about Apple Developer Program role permissions [here](https://developer.apple.com/help/app-store-connect/reference/role-permissions/).
+4. Click Generate. Once the key is generated, it will be added to the list of active keys.
```

Comment:
As quoatation marks are used with +, I'd used them for other buttons too.
```suggestion
4. Click "Generate". Once the key is generated, it will be added to the list of active keys.
```